### ISSUE TITLE
Missing "use File::Temp" in t/dbdpg_test_setup.pl cause tests to fail

### ISSUE THREAD

**Issue by jackdeguest** [bucardo/dbdpg#83]
When running tests, I get the following output:

```
t/02attribs.t .......... Please wait, creating new database (version 12.8) for testing
Undefined subroutine &File::Temp::tempdir called at t/dbdpg_test_setup.pl line 376.
t/02attribs.t .......... Dubious, test returned 1 (wstat 256, 0x100)
```
which eventually leads the tests to fail.
I have added `use File::Temp` at the top of your script `t/dbdpg_test_setup.pl`, performed the tests again, and it worked normally.

**Comment by ilmari** [comment]
Duplicate of #79, fixed in b25ecc46de32b6bbe955f32d6dbd1b95c58ac755.

**Comment by toddbruner** [comment]
This bug is still present in 3.15.0 that is currently the one available on CPAN
